apiserver/apiserver.go

```diff
@@ -117,7 +117,7 @@ type Server struct {
 // LoginValidator functions are used to decide whether login requests
 // are to be allowed. The validator is called before credentials are
 // checked.
-type LoginValidator func(params.LoginRequest) error
+type LoginValidator func(authUser string) error
```

Comment:
I wanted to narrow the scope of the arg passed to validate. None of the implementations used anything other than req.AuthTag, so why pass the entire request struct and allow unnecessary coupling. Case in point - see maintenanceInProgress() where they wanted to pass in a user but had to construct a whole request struct. I've changed the func to take a tag.